Kannst Du mir bitte da nochmal auf die Sprünge helfen? 
make bricht ab mit

uli@UrsaMajor:~/evcc$ make
Version: 0.111.1 21eb5640 2023-01-22_14:30:19
CGO_ENABLED=0 go build -v -tags=release -ldflags='-X github.com/evcc-io/evcc/server.Version=0.111.1 -X github.com/evcc-io/evcc/server.Commit=21eb5640 -s -w'
main.go:17:13: pattern dist: no matching files found
make: *** [Makefile:72: build] Fehler 1

--- a/server/influxdb.go
+++ b/server/influxdb.go
@@ -101,6 +101,11 @@ func (m *Influx) Run(loadPoints []loadpoint.API, in <-chan util.Param) {
 			}
 
 		default:
+			// allow writing nil values
+			if param.Val == nil {
+				break
+			}
+
 			// slice of structs
 			if typ := reflect.TypeOf(param.Val); typ.Kind() == reflect.Slice && typ.Elem().Kind() == reflect.Struct {
 				val := reflect.ValueOf(param.Val)
@@ -125,14 +130,9 @@ func (m *Influx) Run(loadPoints []loadpoint.API, in <-chan util.Param) {
 						writer.WritePoint(p)
 					}
 				}
-
-				continue
 			}
 
-			// any other unsupported type
-			if param.Val != nil {
-				continue
-			}
+			continue
 		}
 
 		// write asynchronously

--- a/server/influxdb.go
+++ b/server/influxdb.go
@@ -11,6 +11,7 @@ import (
 	"github.com/evcc-io/evcc/core/loadpoint"
 	"github.com/evcc-io/evcc/util"
 	influxdb2 "github.com/influxdata/influxdb-client-go/v2"
+	"github.com/influxdata/influxdb-client-go/v2/api"
 	influxlog "github.com/influxdata/influxdb-client-go/v2/log"
 )
 
@@ -57,6 +58,12 @@ func NewInfluxClient(url, token, org, user, password, database string) *Influx {
 	}
 }
 
+// writePoint asynchronously writes a point to influx
+func (m *Influx) writePoint(writer api.WriteAPI, key string, fields map[string]any, tags map[string]string) {
+	m.log.TRACE.Printf("write %s=%v (%v)", key, fields, tags)
+	writer.WritePoint(influxdb2.NewPoint(key, tags, fields, time.Now()))
+}
+
 // Run Influx publisher
 func (m *Influx) Run(loadPoints []loadpoint.API, in <-chan util.Param) {
 	writer := m.client.WriteAPI(m.org, m.database)
@@ -82,7 +89,7 @@ func (m *Influx) Run(loadPoints []loadpoint.API, in <-chan util.Param) {
 			}
 		}
 
-		fields := make(map[string]interface{})
+		fields := make(map[string]any)
 
 		tags := make(map[string]string)
 		if param.Loadpoint != nil {
@@ -124,10 +131,7 @@ func (m *Influx) Run(loadPoints []loadpoint.API, in <-chan util.Param) {
 						fields["value"] = v
 						tags["id"] = strconv.Itoa(i + 1)
 
-						// write asynchronously
-						m.log.TRACE.Printf("write %s=%v (%v)", key, fields, tags)
-						p := influxdb2.NewPoint(key, tags, fields, time.Now())
-						writer.WritePoint(p)
+						m.writePoint(writer, key, fields, tags)
 					}
 				}
 			}
@@ -135,10 +139,7 @@ func (m *Influx) Run(loadPoints []loadpoint.API, in <-chan util.Param) {
 			continue
 		}
 
-		// write asynchronously
-		m.log.TRACE.Printf("write %s=%v (%v)", param.Key, fields, tags)
-		p := influxdb2.NewPoint(param.Key, tags, fields, time.Now())
-		writer.WritePoint(p)
+		m.writePoint(writer, param.Key, fields, tags)
 	}
 
 	m.client.Close()

--- a/server/influxdb.go
+++ b/server/influxdb.go
@@ -120,7 +120,7 @@ func (m *Influx) Run(loadPoints []loadpoint.API, in <-chan util.Param) {
 						n := typ.Field(j).Name
 						v := val.Field(j).Interface()
 
-						key := param.Key + strings.ToLower(n[:1]) + n[1:]
+						key := param.Key + strings.ToUpper(n[:1]) + n[1:]
 						fields["value"] = v
 						tags["id"] = strconv.Itoa(i + 1)
 

